kitten.html

```diff
@@ -33,6 +33,7 @@
         <div class="u"></div>
       </div>
       <nav>
+        <div class="content-center"> 
```

Comment:
You can just add the `class` directly to the `<nav>` here:

```suggestion
      <nav class="content-center">
```